blaze-core/src/main/scala/org/http4s/blazecore/util/CachingStaticWriter.scala

```diff
@@ -19,7 +19,9 @@ import scala.concurrent.{ExecutionContext, Future}
 private[http4s] class CachingStaticWriter[F[_]](
     out: TailStage[ByteBuffer],
     bufferSize: Int =
-      8 * 1024)(implicit protected val F: Effect[F], protected val ec: ExecutionContext)
+      8 * 1024)(implicit
```

Comment:
This line break looks a bit weird to me. I wonder was it enforced by scalafmt or simply inherited?

Reply:
I think I like that it broke the second parameter list onto its own line to match the first.  I suspect this one looks weird because of the default argument, which I might prefer it didn't break.

We generally try to adhere to scalafmt defaults, with the exception of vertical alignment, which bloats diffs.  But if there's a setting that makes this look less weird without introducing a lot more noise, I'm open to seeing what it looks like.

Reply:
well, there're some other examples below how scalafmt allows to keep parameter groups separated. I guess, for this sample it could look like:
```
  out: TailStage[ByteBuffer],
  bufferSize: Int = 8 * 1024
)(implicit
  protected val F: Effect[F],
  ...
```
Although not my favorite way of formatting, but looks pretty clear to me.

Reply:
Yes, I like your rendering better than what scalafmt did.